coalib/settings/FunctionMetadata.py

```diff
@@ -102,7 +102,7 @@ def _get_param(param, section, annotation):
 
         try:
             return annotation(section[param])
-        except:
+        except TypeError:
```

Comment:
I don't think that `coala` should be dependant on `coala-bears`, because if it is there is no point in keeping them in separate repositories.